conf/messages

```diff
@@ -446,3 +446,8 @@ error.required=This field is required
 error.email=This email address is invalid
 error.email_acceptable=This email address is not acceptable
 error.email_unique=This email address is already taken
+blindfoldChess=Blindfold chess (invisible pieces)
+moveConfirmation=Move confirmation
+inCorrespondenceGames=In correspondence games
+ifRatingIsPlusMinus500=If rating is ± 500
```

Comment:
`500` should be parameterized. Example above: `maximumNbCharacters=Maximum: %s characters.`